app/store.js

```diff
@@ -1,5 +1,5 @@
 import Store from "./store/Store";
-import rootReducer from "./store/reducers/rootReducer";
+import rootReducer from "./store/state/rootReducer";
```

Comment:
i renamed reducers folder to state
i wish to have selectors and ... inside their scope